src/views/AmalgamationRegular/ReviewConfirm.vue

```diff
@@ -361,4 +370,9 @@ h2::before {
   counter-increment: header-counter;
   content: counter(header-counter) '. ';
 }
+
+// override base class text size
+.message-box p {
+  font-size: $px-16;
+}
```

Comment:
I'm waiting to hear back from Yui on whether I should make this change to the base class (which affects the other message boxes) or just here.

Reply:
I think Yui won't be back until next week Sev.

Reply:
Oh. In that case, let's leave it like this for now.